e2e/demo-po/toast.po.ts

```diff
@@ -0,0 +1,26 @@
+import {BasePO} from '@agnos-ui/base-po';
+import type {Locator} from '@playwright/test';
+
+export class ToastDemoPO extends BasePO {
+	private readonly locators = {
+		addToastButton: '.addToast',
+		positionSelect: 'select[id=positionSelect]',
+		toastContainer: 'div.toast-container',
+	};
+
+	getComponentSelector(): string {
+		return '.container';
+	}
+
+	get locatorAddToastButton(): Locator {
+		return this.locatorRoot.locator(this.locators.addToastButton);
+	}
+
+	get locatorPositionSelect(): Locator {
+		return this.locatorRoot.locator(this.locators.positionSelect);
+	}
+
+	get locatorToastContainer(): Locator {
```

Comment:
This return all the containers. In the associated tests, it seems more useful to target a specific container.

In this case, it would be better to write it as a function instead of a getter:

```typsescript
    locatorToastContainer(index: number): Locator {
        return this.locatorRoot.locator(this.locators.toastContainer).nth(index);
	}
}

```

```typsescript
// Now you can use :
await toastDemoPO.locatorToastContainer(4);

instead of
(await toastDemoPO.locatorToastContainer.all()).at(4);
```